packages/aws-cdk-lib/cloudformation-include/test/test-templates/intrinsic-deletion-policy.json

```diff
@@ -0,0 +1,53 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Transform": "AWS::LanguageExtensions",
+    "Parameters": {
+        "Stage": {
+            "Type": "String",
+            "AllowedValues": [
+                "Prod",
+                "Staging",
+                "Dev"
+            ]
+        }
+    },
+    "Conditions": {
+        "IsProd": {
+            "Fn::Equals": [
+                {
+                    "Ref": "Stage"
+                },
+                "Prod"
+            ]
+        }
+    },
+    "Resources": {
+        "Table": {
```

Comment:
Can we add another resource called i.e. `Table2` that uses `!Ref` intrinsic? The reason being is that `looksLikeCfnIntrinsic` function handles `Ref` different from `Fn::` intrinsics.

Reply:
Done.